analysis/src/Protocol.ml

```diff
@@ -1,6 +1,8 @@
 type position = {line: int; character: int}
 type range = {start: position; end_: position}
 type markupContent = {kind: string; value: string}
+type command = {title: string; command: string}
```

Comment:
Is this a generic command, or only for codelenses?

Reply:
Generic command from the protocol. I'll add a reference to it in the spec.